pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -126,7 +135,7 @@ class AgentNode(BaseNode[GraphAgentState, GraphAgentDeps[DepsT, Any], result.Fin
 
 
 def is_agent_node(
-    node: BaseNode[GraphAgentState, GraphAgentDeps[T, Any], result.FinalResult[S]] | End[result.FinalResult[S]],
+    node: (BaseNode[GraphAgentState, GraphAgentDeps[T, Any], result.FinalResult[S]] | End[result.FinalResult[S]]),
```

Comment:
I'm seeing a number of unrelated changes likely due to your local line length limit, can you please remove those? 

In `pyproject.toml`, we configure a line length of 120:

```
[tool.ruff]
line-length = 120
```

It looks like your local settings are overriding that